helm-chart/templates/ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $name := .Values.name -}}
+{{- $service_port := .Values.service.port -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: {{ $name }}
+  annotations:
+    {{- if .Values.ingress.basic_auth }}
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: carforyou-components-pkg-basic-auth
+    nginx.ingress.kubernetes.io/auth-realm: "Authentication Required - carforyou"
+
+    # WARN: an invalid nginx directive may crash the ingress of the whole kubernetes cluster, please make triple sure your changes are valid
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      # whitelist office IPs (everything else uses basic-auth)
+      satisfy any;
+      allow 77.109.147.226; # bluecat_team (Altstetten)
+      allow 77.109.147.227; # bluecat_guest (Altstetten)
+      allow 178.83.153.152; # CarForYou (Stadelhofen)
```

Comment:
Not sure bout the Altstetten ones but we don't have the Stadelhofen office for a while so I think we can drop that  one

Reply:
woooo good point! I will remove it from the header-footer as well.

Reply:
Maybe we can make another clean up in all the projects when we are fine about removing the ones of  `Altstetten`